counter/src/Frame/Frame.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { Row, Col, Container, ListGroup, Card } from "react-bootstrap"
+
+export const Main = ({items, inner}) =>
+  <Container>
+    <Row><Col>&nbsp;</Col></Row>
+    <Row>
+      <Col md={2}>
+        <ListGroup>
+          {items.map(({title, isSelected, onSelect}) =>
+            <ListGroup.Item active={isSelected} onClick={onSelect} action>
```

Comment:
FYI, most React developers would expect `key` to be required here. The only reason we don’t is because of this: https://github.com/collegevine/purescript-elmish/pull/1/files#r280460579

Reply:
No, that is incorrect. This code does not go through our purescript code anymore, this is only React. Therefore, yes, a key is actually required here. I will add.